hack/generate/samples/internal/ansible/memcached.go

```diff
@@ -76,7 +76,13 @@ func (ma *MemcachedAnsible) Run() {
 	ma.addingAnsibleTask()
 	ma.addingMoleculeMockData()
 
-	ma.ctx.CreateBundle()
```

Comment:
The goal here is to centralize the implementation to allow samples and tests to use the same. Then, the `StripBundleAnnotations` is only required for the samples.